File: 백준/Bronze/2588. 곱셈/곱셈.c
#include<stdio.h>

int main(void){
    int a,b;
    int line3,line4,line5,line6;
    scanf("%d %d", &a, &b);
    line3 = b%10*a;
    line4 = (b%100)/10*a;
    line5 = (b/100)*a;
    line6 = a*b;
    
    printf("%d\n",line3);
    printf("%d\n",line4);
    printf("%d\n",line5);
    printf("%d",line6);
}

File: 백준/Bronze/2588. 곱셈/곱셈.py
a = int(input())
b = int(input())

line3 = b%10*a
line4 = int(int(b%100)/10)*a
line5 = int(b/100)*a
line6 = a*b

print(line3)
print(line4)
print(line5)
print(line6)